ads/inabox/inabox-messaging-host.js

```diff
@@ -197,14 +191,13 @@ export class InaboxMessagingHost {
    *
    * @param {!Object} request
    * @param {!Window} source
-   * @param {string} origin
    * @param {?JsonObject} data
    */
-  sendPosition_(request, source, origin, data) {
+  sendPosition_(request, source, data) {
     dev().fine(TAG, 'Sent position data to [%s] %s', request.sentinel, data);
     source./*OK*/ postMessage(
       serializeMessage(MessageType.POSITION, request.sentinel, data),
-      origin
+      '*'
```

Comment:
This works well as long as we have the request sentinel. The only concern as @lannka mentioned is that the sentinel will be shared to other origins. LGTM as long as we are good with that.